search/materialize_test.go

```diff
@@ -0,0 +1,163 @@
+package search
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/parquet-go/parquet-go"
+	"github.com/prometheus/prometheus/model/labels"
+	"github.com/prometheus/prometheus/storage"
+	"github.com/prometheus/prometheus/tsdb/chunkenc"
+	"github.com/prometheus/prometheus/util/teststorage"
+	"github.com/stretchr/testify/require"
+	"github.com/thanos-io/objstore/providers/filesystem"
+
+	"github.com/prometheus-community/parquet-common/convert"
+	"github.com/prometheus-community/parquet-common/schema"
+	"github.com/prometheus-community/parquet-common/util"
+)
+
+func TestMaterializeE2E(t *testing.T) {
```

Comment:
I like this test and the example usage you shared in the description of the PR of how to use this Materialize() "API".

I'm suggesting a slight reordering here if you'd like to take it in https://github.com/alanprot/parquet-common/pull/3

Reply:
Thanks a lot! :D

Its way better now!!! :D